.github/workflows/ci.yml

```diff
@@ -1,18 +1,19 @@
 name: CI
 
-on: 
-  pull_request:
+on:
```

Comment:
The previous configuration would not trigger CI when a pull request was merged to the default branch.